modules/nw-cluster-mtu-change.adoc

```diff
@@ -347,20 +349,24 @@ where `<config_name>` is the name of the machine config from the `machineconfigu
 If the machine config is successfully deployed, the previous output contains the `/etc/NetworkManager/conf.d/99-<interface>-mtu.conf` file path and the `ExecStart=/usr/local/bin/mtu-migration.sh` line.
 endif::local-zone,wavelength-zone,post-aws-zones,outposts[]
 
-. To finalize the MTU migration, enter the following command for the OVN-Kubernetes network plugin:
+. Finalize the MTU migration for your plugin. In both example commands, ` <mtu>` specifies the new cluster network MTU that you specified with `<overlay_to>`.
 +
-[source,terminal]
+.. To finalize the MTU migration, enter the following command for the OVN-Kubernetes network plugin:
```

Comment:
If OVN-Kubernetes vs. OpenShift SDN are two options, these subsetps should probably be unordered list items (do one or the other) rather than ordered list items (do one and then also the other).
```suggestion
** To finalize the MTU migration, enter the following command for the OVN-Kubernetes network plugin:
```

Reply:
Yes, bullet points make more sense!